docs/source/cupy-reference/difference.rst

```diff
@@ -0,0 +1,55 @@
+Difference between CuPy and NumPy
+=================================
+
+The interface of CuPy is designed to obey the interface of NumPy.
+However it has some differenece.
+
+
+Cast behavior from float to integer
+-----------------------------------
+
+Some casting behavior from float to integer is not defined.
+For example from a negative float to unsigned integer and infinity to integer.
+It depends on your CPU.
```

Comment:
You are trying to describe two different things simultaneously, which makes me confused a bit. One is the difference of NumPy and CuPy. The other is that the behavior of NumPy itself can change depending on the CPU to use. So, why don't you explain them separately?